src/extension.ts

```diff
@@ -854,6 +856,17 @@ class ExtensionManager implements vscode.Disposable {
   resetState(folder?: vscode.WorkspaceFolder) { return this.mapCMakeToolsFolder(cmt => cmt.resetState(), folder); }
 
   async viewLog() { await logging.showLogFile(); }
+
+  async hideLaunchCommand(shouldHide: boolean = true) {
+    // Don't hide command to launch/debug target here since the target can still be useful, one example is in launch.json
```

Comment:
I don't understand this comment.  You are hiding the launch target command with the same "context" state.

Reply:
Typo... Should be get/set launch/debug target